### PR TITLE
Fix references to CommCareCaseGroup

### PR DESCRIPTION
**Pull request by gcapalbo**
code buddy @NoahCarnahan 
fyi @dannyroberts

### CODE REVIEW

File: corehq/apps/reminders/util.py [hunk 1]
```diff
@@ -95,12 +95,6 @@ def get_form_name(form_unique_id):
     return app.name + "/" + module_name + "/" + form_name
 
 def get_recipient_name(recipient, include_desc=True):
-    # Circular imports
-    from corehq.apps.users.models import CouchUser
-    from corehq.apps.groups.models import Group
-    from casexml.apps.case.models import CommCareCase
-    from casexml.apps.case.models import CommCareCaseGroup
```
Comment (gcapalbo) [inline corehq/apps/reminders/util.py]: found this reference still pointing to the old location of CommCareCaseGroup, and then realized that none of these imports have a circular dependency anymore

**Comment by dannyroberts** [comment]
Nice, good to merge when the tests pass (though I don't know why this would make them fail, if you want to merge more urgently)

**Comment by gcapalbo** [comment]
@dannyroberts yeah this wouldn't cause any tests to fail and we should merge this before deploy, so if you could merge now that would be great

**Comment by dannyroberts** [comment]
:+1:

**Pull request merged**